jest.setup.ts

```diff
@@ -1 +1,6 @@
 import '@testing-library/jest-dom/extend-expect'
+import { configure } from '@testing-library/dom'
+
+configure({
+    testIdAttribute: 'id',
```

Comment:
This configures the `ByTestId` query APIs to look for an `id` attribute selector instead of `data-testid`